<ui version="4.0" >
 <class>NetworkEditDlg</class>
 <widget class="QDialog" name="NetworkEditDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Edit Network Settings</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="3" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Identity:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Network:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Group:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QComboBox" name="networkGroup" >
       <property name="editable" >
        <bool>true</bool>
       </property>
       <property name="insertPolicy" >
        <enum>QComboBox::NoInsert</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QComboBox" name="identityList" />
       </item>
       <item>
        <widget class="QPushButton" name="editIdentities" >
         <property name="sizePolicy" >
          <sizepolicy>
           <hsizetype>4</hsizetype>
           <vsizetype>0</vsizetype>
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text" >
          <string>Edit...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="networkName" />
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="networkDesc" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Comment:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="enableAutoConnect" >
     <property name="text" >
      <string>Auto-connect on startup</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="currentIndex" >
      <number>1</number>
     </property>
     <widget class="QWidget" name="serversTab" >
      <property name="enabled" >
       <bool>true</bool>
      </property>
      <attribute name="title" >
       <string>Servers</string>
      </attribute>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>9</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QListWidget" name="serverList" />
       </item>
       <item>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QPushButton" name="addServer" >
           <property name="text" >
            <string>&amp;Add...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="editServer" >
           <property name="text" >
            <string>&amp;Edit...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteServer" >
           <property name="text" >
            <string>De&amp;lete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="upServer" >
           <property name="text" >
            <string>Move &amp;Up</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="downServer" >
           <property name="text" >
            <string>Move &amp;Down</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="performTab" >
      <attribute name="title" >
       <string>Perform</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>8</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QTextEdit" name="performEdit" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="lineWrapMode" >
            <enum>QTextEdit::NoWrap</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4" >
           <property name="text" >
            <string>These commands are executed after connect.</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" native="1" name="layoutWidget" />
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NetworkEditDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>216</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel" >
     <x>216</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NetworkEditDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>216</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel" >
     <x>216</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
